specification/iothub/resource-manager/Microsoft.Devices/preview/2019-03-22-preview/iothub.json

```diff
@@ -2534,6 +2541,33 @@
         "isEnabled"
       ]
     },
+    "EnrichmentProperties": {
+      "description": "The properties of an enrichment that your IoT hub applies to messages to endpoints.",
+      "type": "object",
+      "properties": {
+        "key": {
+          "description": "The key for the enrichment property.",
+          "type": "string"
+        },
+        "value": {
+          "description": "The value for the enrichment property.",
```

Comment:
Do we want to document reserved keywords here? or point to documentation?

Reply:
@ash2017 ?

Reply:
We can do this in the doc...and per above comment, link to it.